### ISSUE TITLE
Add more aerial image providers

### ISSUE THREAD

**Issue by Binnette** [tpetillon/roofmapper#1]
Hi can you please add more image providers?

![image](https://user-images.githubusercontent.com/6597721/58374550-717a8480-7f40-11e9-9ab7-847c18005b95.png)

The [iD editor](https://github.com/openstreetmap/iD) is giving a lot of image providers.

Here are my favorites:
1. BDOrtho: for France (a must have! images are in high definition!)
2. Digital Global
3. Esri

Add as much image providers as you can, because image quality is not the same everywhere.

The complete list from iD editor:

![image](https://user-images.githubusercontent.com/6597721/58374543-5d368780-7f40-11e9-9814-b112cd2f6323.png)

**Comment by cquest** [comment]
Higher resolution imagery helps to determine roof type. The current one is a guess games in many cases.

**Comment by tpetillon** [comment]
I totally agree! The current imagery layer is inadequate in many places. Having a selection of providers is needed.

Originally, this project was little more than an experiment, and adding new features was cumbersome. But because I'd like to turn this tool into something more useful (with more usecases), I have started to remake it with more modern libraries.

When the new version becomes usable, I will probably ask you, @cquest, for help on how to display IGN's BD Ortho. (If I remember correctly, it has to go through a proxy on osm-fr servers.)
